src/WebJobs.Script/Utility.cs

```diff
@@ -520,8 +520,28 @@ private static bool ContainsFunctionWithWorkerRuntime(IEnumerable<FunctionMetada
             return false;
         }
 
-        public static bool CheckAppOffline(string scriptPath)
+        public static async Task MarkContainerDisabled()
         {
+            var disableContainerFilePath = Path.Combine(Path.GetTempPath(), ScriptConstants.DisableContainerFileName);
```

Comment:
Is the temp directory durable enough for your purposes?

Reply:
Yes. It is good enough.

Reply:
Yes.  It works based on my testing.